media/css/mozorg/try-picture-in-picture.scss

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$brand-theme: "firefox";
+
+@import "../firefox/whatsnew/includes/base";
```

Comment:
Should we copy these styles into this page's stylesheet so it's truly a standalone page?